### ISSUE TITLE
improving amp-image-lightbox for desktop

### ISSUE THREAD

**Issue by 2shrestha22** [ampproject/amphtml#24531]
## Describe the new feature or change to an existing feature you'd like to see

When AMP is used in desktop version the amp-image-lightbox effect cause small images to stretch. Images tends to fill the whole screen but in desktop this feature makes images ugly. Is there any way not to exceed width and height of images?

## Describe alternatives you've considered

Provide a clear and concise description of any alternative solutions or features you've considered.

**Comment by Stefany93** [comment]
^^ I think the whole point of the lightbox is to provide a higher resolution image to the user on click. so you need to supply a higher resolution pic and if not, you don't need a lightbox. 

I could be wrong tho.

**Comment by 2shrestha22** [comment]
Yes it is. But the things is that **lightbox is enabled by default** in all images including featured image and small screenshots images that I use. Any way to avoid auto enabled lightbox?

**Comment by Stefany93** [comment]
Are you sure you are not talking about `amp-image-gallery` ? With `amp-img-lightbox` you have to manually specify which images to be lightboxed.

**Comment by 2shrestha22** [comment]
Yes it was not happening before. I was manually enabling lightbox effect while editing post. Here is my site - https://sangams.com.np/

**Comment by alanorozco** [comment]
@2shrestha22

[You can disable auto lightbox](https://github.com/ampproject/amphtml/blob/master/spec/auto-lightbox.md) by setting the `data-amp-auto-lightbox-disable` on any element. Set it on `<body>` to disable for the whole document.

Additionally, if you include the `amp-lightbox-gallery` extension and use it explicitly, auto-lightbox won't apply.

**Comment by 2shrestha22** [comment]
@alanorozco 

Thanks for the solution. Is it possible to makes image not fill the width and height instead show actual size of image.

**Comment by cathyxz** [comment]
@2shrestha22 I think that's probably a better use case for `amp-lightbox` than for `amp-image-lightbox` or `amp-lightbox-gallery`. Would styling your image for `amp-lightbox` work in this case?
